src/app/models/register/register-details.ts

```diff
@@ -0,0 +1,11 @@
+export class Registerdetails {
```

Comment:
Good work @akram256, we could use PascalCase for all our classes, Just to maintain the customary JS conventions.
You could take a look here https://github.com/landvilleng/landville-frontend/pull/15#pullrequestreview-265981361

Reply:
Worked on this. Thanks @CryceTruly